src/main/java/com/feywild/feywild/entity/goals/ReturnToPositionKnightGoal.java

```diff
@@ -0,0 +1,33 @@
+package com.feywild.feywild.entity.goals;
+
+
+import com.feywild.feywild.entity.base.IAngry;
+import net.minecraft.world.entity.Mob;
+import net.minecraft.world.phys.Vec3;
+
+import java.util.function.Supplier;
+
+public class ReturnToPositionKnightGoal extends  GoToTargetPositionGoal{
+    private Mob entity;
```

Comment:
This can use a more specific type in the `entity` field I think. You should make it a `BeeKnight`